libcxx/test/libcxx/memory/shared_ptr_array.pass.cpp

```diff
@@ -10,10 +10,6 @@
 // UNSUPPORTED: c++03, c++11, c++14, c++17
 
 // These compiler versions and platforms don't enable sized deallocation by default.
-// ADDITIONAL_COMPILE_FLAGS(clang-17): -fsized-deallocation
-// ADDITIONAL_COMPILE_FLAGS(clang-18): -fsized-deallocation
-// ADDITIONAL_COMPILE_FLAGS(apple-clang-15): -fsized-deallocation
-// ADDITIONAL_COMPILE_FLAGS(apple-clang-16): -fsized-deallocation
```

Comment:
I think you'll need `ADDITIONAL_COMPILE_FLAGS(apple-clang-17): -fsized-deallocation` since AppleClang still doesn't enable it by default.